pipdb.js

```diff
@@ -0,0 +1,114 @@
+require('buffertools').extend();
+
+var dbIndices = {};
+var dbTypes = {};
+
+var dbEntryType = ['Bool', 'Int8', 'UInt8', 'Int32', 'UInt32', 'Float', 'String', 'List', 'Dictionary'];
+
```

Comment:
I wasn't sure how you'd prefer to split up the functionality between the two projects. Feel free to split this up and move it around however you deem necessary.